lib/workers/repository/reconfigure/utils.ts

```diff
@@ -1,3 +1,70 @@
+import is from '@sindresorhus/is';
+import JSON5 from 'json5';
+import type { RenovateConfig } from '../../../config/types';
+import { logger } from '../../../logger';
+import { platform } from '../../../modules/platform';
+import { scm } from '../../../modules/platform/scm';
+import type { BranchStatus } from '../../../types/branch-status';
+import { readLocalFile } from '../../../util/fs';
+import { detectConfigFile } from '../init/merge';
+
 export function getReconfigureBranchName(prefix: string): string {
   return `${prefix}reconfigure`;
 }
+
+export async function setBranchStatus(
```

Comment:
can you extract this refactor (function move) to a separate pr?

Reply:
https://github.com/renovatebot/renovate/pull/35478